src/main/java/org/folio/consortia/client/PermissionsClient.java

```diff
@@ -16,4 +18,7 @@ public interface PermissionsClient {
 
   @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
   PermissionUser create(@RequestBody PermissionUser permissionUser);
+
+  @PostMapping(value = "/{userId}/permissions?indexField=userId", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
its simple string only ,
And i think its been used to avoid confusion while making **_WHERE_** clause for query. because we are just sending UUID in request path and it is difficult to identify if it is userID or just id of permission.


{
  **"id": "01e31f75-4a96-4367-91b9-b2a1af7e56fa",
  "userId": "2ee07dc7-835f-4a33-a783-db2ae3f1238c",**
  "metadata": {
    "createdDate": "2023-05-19T04:53:58.880Z",
    "updatedDate": "2023-05-19T04:53:58.880Z"
  },
  "permissions": []
}